library/src/main/kotlin/com/connectrpc/protocols/GRPCWebInterceptor.kt

```diff
@@ -244,7 +244,14 @@ internal class GRPCWebInterceptor(
             if (i > 0) {
                 val name = line.substring(0, i).trim()
                 val value = line.substring(i + 1).trim()
-                trailers[name.lowercase()] = listOf(value)
```

Comment:
This failed to merge values, both if there were multiple values for a single key and if there were conflicts after lower-casing. Instead of "last one wins" with a singular value, we merge the values.